Sources/NIO/Selector.swift

```diff
@@ -139,22 +326,8 @@ final class Selector<R: Registration> {
 #endif
     }
 
-#if os(Linux)
-
-    private static func toEpollEvents(interested: IOEvent) -> UInt32 {
-        // Also merge EPOLLRDHUP in so we can easily detect connection-reset
-        switch interested {
-        case .read:
-            return Epoll.EPOLLIN.rawValue | Epoll.EPOLLERR.rawValue | Epoll.EPOLLRDHUP.rawValue
-        case .write:
-            return Epoll.EPOLLOUT.rawValue | Epoll.EPOLLERR.rawValue | Epoll.EPOLLRDHUP.rawValue
-        case .all:
-            return Epoll.EPOLLIN.rawValue | Epoll.EPOLLOUT.rawValue | Epoll.EPOLLERR.rawValue | Epoll.EPOLLRDHUP.rawValue
-        case .none:
-            return Epoll.EPOLLERR.rawValue
-        }
-    }
 
+#if os(Linux)
```

Comment:
This looks silly: we know what our queue supported platforms are, we should just write them down.

Reply:
fixed, left over from old when there was code in the if